cli/module_loader.rs

```diff
@@ -1464,10 +1467,14 @@ impl<TGraphContainer: ModuleGraphContainer> NodeRequireLoader
           ModuleKind::Cjs,
           &text.into(),
         )
-        .map(Cow::Owned)
-        .map_err(JsErrorBox::from_err)
+        .map_err(JsErrorBox::from_err)?;
+      self.parsed_soure_cache.free(&specifier);
```

Comment:
It's better to do this at a higher level so the caller can decide when to free these from the cache.